src/cpp/desktop/DesktopOptions.cpp

```diff
@@ -266,7 +266,17 @@ QString Options::proportionalFont() const
    // quotes; e.g. "sans-serif" is a signal to look for a font called sans-serif
    // rather than use the default sans-serif font!
    if (selectedFont == sansSerif)
+   {
+#if defined(__APPLE__)
+      // Qt 5.15 has a known issue on MacOS in which it can't detect any fonts; if this happens,
```

Comment:
Is there an associated QTBUG we could link to here?

Reply:
No, I couldn't find one.

Reply:
Should this be happening here too (or instead)?

https://github.com/rstudio/rstudio/pull/8142/files#diff-bbf9a277ef640cbca697c4b529194712a2a7f42175e76a30dcaff778a6545b38

Reply:
Oh, I found the Qt bug! https://bugreports.qt.io/browse/QTBUG-87267

Reply:
@gtritchie No, that's the last resort font and it's okay if it's a system font. This is the font used for `proportionalFont` everywhere in our UI.

Based on my reading of 87267, we couldn't even bet on this change making it into Qt 5.15.2 ("fix versions: none"). Is that right?

Reply:
> Based on my reading of 87267, we couldn't even bet on this change making it into Qt 5.15.2 ("fix versions: none"). Is that right?

Unclear from the Qt issue; I've asked in the issue.

Reply:
Qt issue was updated with: “The fix is already in Qt 6 beta-1 which was released on Tuesday. It will also be a part of Qt 5.15.2 which is planned for release in November.”